packages/sit-onyx/src/components/OnyxFileUpload/OnyxFileUpload.vue

```diff
@@ -59,7 +73,32 @@ const modelValue = useVModel<TModelValue, "modelValue", typeof props, undefined>
 
 const input = useTemplateRef<HTMLInputElement>("inputRef");
 
-const currentFiles = computed<File[]>(() => asArray(modelValue.value));
+const currentFiles = computed<File[]>(() => asArray(modelValue.value ?? []) as unknown as File[]);
+const hideFiles = ref(false);
+
+const fileStatuses = computed((): (FileCardStatus | undefined)[] => {
```

Comment:
We should use `useFileSize` for formatting the size inside the translated message so they are human readable strings instead of big binary numbers